tests/compilation/passingContracts.ts

```diff
@@ -34,9 +34,9 @@ export const passingContracts = [
   // 'tests/compilation/contracts/deleteUses.sol',
   'tests/compilation/contracts/enums.sol',
   // 'tests/compilation/contracts/enums7.sol',
-  // 'tests/compilation/contracts/errorHandling/assert.sol',
-  // 'tests/compilation/contracts/errorHandling/require.sol',
-  // 'tests/compilation/contracts/errorHandling/revert.sol',
+  'tests/compilation/contracts/errorHandling/assertHandling.sol',
+  'tests/compilation/contracts/errorHandling/requireHandling.sol',
+  'tests/compilation/contracts/errorHandling/revertHandling.sol',
```

Comment:
This test is failing due to errors related to input checks which were being solved in a separate PR https://github.com/NethermindEth/warp/pull/1041
Let's comment this test case here and uncomment it once the error is handled